lib/challenge.js

```diff
@@ -253,13 +328,18 @@ function prompt11() {
  */
 
 function prompt12() {
-  // YOUR CODE HERE
+  // for (let i = 1; i < 11; i++) {
+  //   for (let j = 11; j < 21; j++) {
+  //     console.log(`i: ${i} / j: ${j}`);
+  //   }
+  // }
 }
 
 // COMMENT IN THE BELOW LINE OF CODE TO TEST YOUR OUTPUT IN THE BROWSER!
 // prompt12();
 
 // 🌟 MAKE A COMMIT: "Complete prompt 12"
+// Found it interesting that the code above did not output the integers for i and j with a 1:1 ratio, i.e., that I did not get 1/11, 2/12, 3/13, etc... Experimented with adjusting the terminating condition for the j loop to see if I could achieve something more 1:1. The closest I got was when I set the j loop terminating condition to (j < i + 11).
 
```

Comment:
I think you would be able to accomplish this by adding an `if` condition within your loops.  Try it out and see if you can get it!